src/apis/AmenityAPI.jsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright © 2023, School CRM Inc. ALL RIGHTS RESERVED.
+ *
+ * This software is the confidential information of School CRM Inc., and is licensed as
+ * restricted rights software. The use, reproduction, or disclosure of this software is subject to
+ * restrictions set forth in your license agreement with School CRM.
+ */
+
+import { api } from "./config/axiosConfig";
+import { defineCancelApiObject } from "./config/axiosUtils";
+import { Utility } from "../components/utility";
+
+const { getLocalStorage } = Utility();
+
+export const AmenityAPI = {
+    /** Get amenities from the database that meets the specified query parameters
+     */
+    getAll: async (conditionObj = false, page = 0, size = 5, search = false, authInfo, cancel = false) => {
```

Comment:
proper naming convention, either use amenity in all the api or in none.
like getAllAmeneties, createAmenity, updateAmenity.
or
getAll, create, update
follow one rule and modify for all the api files